pandas/core/indexes/base.py

```diff
@@ -2005,15 +2005,35 @@ def __contains__(self, key):
             return False
 
     _index_shared_docs['contains'] = """
-        return a boolean if this key is IN the index
+        Return a boolean if this key is in the index.
 
         Parameters
         ----------
-        key : object
+        key : label
+            The key can be of the same type as the label of :class: `Index`,
```

Comment:
I think the space after `:class:` is not needed. Did you check if this is rendered correctly in the html version? You should be able to generate this page with `./doc/make.py html --single=pandas.Index.contains`

Reply:
Thanks, it renders correctly after removing the space after `:class:`